src/virtual-repeat.js

```diff
@@ -454,7 +457,7 @@ export class VirtualRepeat extends AbstractRepeater {
   }
 
   _calcInitialHeights(itemsLength: number): void {
-    if (this._viewsLength > 0 && this._itemsLength === itemsLength || itemsLength <= 0) {
+    if (this._viewsLength > 0 && this._itemsLength === itemsLength || this._viewsLength > 0 && itemsLength < 0) {
```

Comment:
How could `itemsLength` be less then 0?

Reply:
I just didn't want to break previous logic which prevented less and equal to 0. But now 0 is permitted when there are still views and the height has to be calculated.

Reply:
Sorry I was not able to provide timely feedback. My work on Aurelia happens outside my primary job (and occasionally, when interests combine, during my job). Recently, my time outside my job has been minimized (still have a family and house to attend to).  
This line seems to cause an issue when your list truly has 0 elements. Previously, if we had 0 items, we would not calculate heights because there are no elements to calculate. Modifying this if block like this removes that check. 
`this._viewsLength` doesn't get set until later on in the checkInitialHeights, so initially it will always be 0. Your check needs to account for this case.